### PR TITLE
change(.travis): change travis file to be consistent with package det…

### PR DESCRIPTION
**Pull request by drshrey**
…ails

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+sudo: false
 python:
   - "3.5"
   - "3.4"
   - "2.7"
 
-before_install:
-  - pip install codecov
+before_script:
+  - flake8 verification/ --max-line-length=120
```
Comment (glemmaPaul) [inline .travis.yml]: Flake8 uses the config stated in setup.conf, see an example here: https://github.com/Fueled/django-push-notifications/blob/master/setup.cfg

That setup file basically makes the length not a validation rule.

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+  directories:
+  - $HOME/.pip-cache/
 
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -r requirements-test.txt
+install:
+  - pip install -r requirements-test.txt --allow-all-external --download-cache $HOME/.pip-cache
 
-# command to run tests using coverage, e.g. python setup.py test
-script: coverage run --source verification runtests.py
```
Comment (glemmaPaul) [inline .travis.yml:17]: I rather want to use coverage in this one, so please revert this :) (don't forget to add coverage to requirements-test)

File: .travis.yml (hunk 1 above)
Comment (drshrey) [inline .travis.yml]: Makes sense, understood, will implement

File: verification/__init__.py [hunk 3]
```diff
@@ -1,2 +1,2 @@
 __version__ = '0.1.0'
-# -*- coding: utf-8 -*
\ No newline at end of file
+# -*- coding: utf-8 -*
```
Comment (glemmaPaul) [inline verification/__init__.py]: If you want to define the encoding it should be on the top of the file

**Comment by glemmaPaul** [comment]
Some minor comments, but besides that good work!

File: verification/__init__.py (hunk 3 above)
Comment (drshrey) [inline verification/__init__.py]: sounds good, will do

File: .travis.yml (hunk 2 above)
Comment (drshrey) [inline .travis.yml:17]: sounds good

**Pull request merged**